cms/static/js/views/course_info_handout.js

```diff
@@ -50,10 +50,14 @@ define(['js/views/baseview', 'codemirror', 'common/js/components/views/feedback_
             },
 
             onSave: function(event) {
+                var dataToBeSaved = this.$codeMirror.getValue();
                 $('#handout_error').removeClass('is-shown');
                 $('.save-button').removeClass('is-disabled').attr('aria-disabled', false);
                 if ($('.CodeMirror-lines').find('.cm-error').length == 0) {
-                    this.model.set('data', this.$codeMirror.getValue());
+                    if (dataToBeSaved === '') {
```

Comment:
we can do this way also, I was just concerned about making a generic method that can be used for any other string we need to format.